app/src/main/java/org/coepi/android/ui/cen/CENViewModel.kt

```diff
@@ -17,6 +17,7 @@ class CENViewModel(
     private val bleManager: BleManager
 ) : ViewModel() {
 
+    val curcen = android.util.Base64.encode(repo.CEN.value,0);
```

Comment:
Can you please use imports?

`import android.util.Base64`

Then `Base64.encode` instead of `android.util.Base64.encode`

Reply:
As you've reviewed in PR #18 I'm using android.util.Base64 otherwise it doesn't compile for my mobile (minsdk=21)  Base64 that's not working on my mobile  Base64.Encoder requires API 26, https://developer.android.com/reference/java/util/Base64.Encoder

Reply:
I've changed